### ISSUE TITLE
Confusing blank response for `short search` with no query

### ISSUE THREAD

**Issue by ndbroadbent** [shortcut-cli/shortcut-cli#291]
Hello, I've just installed the shortcut CLI, and I want to list all stories in my workspace. I like the idea of setting up some aliases so I can easily start new stories and checkout a git branch. I'm just having a problem listing stories, because the output is blank if I don't provide a search query. It does work if I provide a query, e.g.:

```
$ short s -s "In Development" Google
#1220 Sign in / sign up with OAuth: Google
...
```

But if I just run `$ short s -s "In Development"`, I get no results:

```
$ short s -s "In Development"
Fetching all stories for search since no search operators were passed ...
$
```

I'd like to be able to list all stories for a given state. I've tried to play around with the `-S` option to "save" a search but that didn't seem to help.

It does work if I run `short s "is:started"` or `short s "state:500000006"`. So I'm not too sure what's the difference between `-s "In Development"` as an argument vs. `state:500000006` as a search operator, but they produce different results.

-----

### Expected Behavior

Show all stories

### Actual Behavior

Shows no stories

### Steps to Reproduce

```
$ short search
Fetching all stories for search since no search operators were passed ...
# (no output)
$ 
```

**Comment by j-martin** [comment]
Hi @ndbroadbent thanks for the report. 

This is most likely a regression I introduced in 3.0.0 or 3.1.0.

**Comment by j-martin** [comment]
Fixed in `3.2.1`.
